app/human-detector-app/package.json

```diff
@@ -22,27 +22,36 @@
     ]
   },
   "dependencies": {
+    "@expo/ngrok": "^2.5.0",
     "@expo/vector-icons": "^13.0.0",
+    "@react-keycloak/native": "^0.6.2",
     "@react-navigation/bottom-tabs": "^6.0.5",
     "@react-navigation/native": "^6.0.13",
     "@react-navigation/native-stack": "^6.1.0",
     "axios": "^0.27.2",
+    "base64url": "^3.0.1",
+    "crypto-js": "^4.1.1",
+    "crypto-ts": "^1.0.2",
```

Comment:
It looks like you're using `crypto-js` and you have  `@types/crypto-js` for TS types, is `crypto-ts` used at all?

Reply:
removed crypto-ts because nah that's not used in my final code 8ccb291158e2ffb91db73d10a7a244967760396d

also removed base64url for the same reason: 52fbbae4c8fd54a1887f57d0ed2fa01fc16e2dab